src/ruby/yast/builtins.rb

```diff
@@ -117,26 +116,26 @@ def self.isempty object
     # - Maps an operation onto all elements key/value and create a list
     # - Maps an operation onto all elements of a list and thus creates a new list.
     # @deprecated use ruby native method {::Enumerable#map}
-    def self.maplist object, &block
+    def self.maplist(object, &block)
       case object
       when ::Array
         res = []
         begin
           object.each do |i|
-            res << block.call(Yast::deep_copy(i))
+            res << block.call(Yast.deep_copy(i))
           end
-        rescue Yast::Break
-          #break skips out of each loop, but allow to keep previous results
+        rescue Yast::Break # rubocop:disable Lint/HandleExceptions
```

Comment:
ugh, this breaks my internal rule to not add tool specific stuff to code as it decrease readability of code. If needed you can add debug log there.